packages/odyssey/src/scss/components/_checkbox-context.scss

```diff
@@ -15,7 +15,7 @@
 .ods-table {
   .ods-checkbox--label {
     &:empty {
-      height: calc(1em * #{$heading-line-height});
```

Comment:
Should we create some variables for these optical spacing units?

Reply:
I *think* its only used in one place at the moment, would hold off until there's a need elsewhere?

Reply:
solved by our new/old units!